spec/features/esri_viewer_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'spec_helper'
+
+feature 'feature_layer reference' do
+  scenario 'displays leaflet viewer', js: true do
+    visit catalog_path('minnesota-test-neighborhoods-pdx')
+    expect(page).to have_css '.leaflet-control-zoom', visible: true
+  end
```

Comment:
Can we also test here that the layer opacity is being setup? There seems to be a bug where the layer opacity indicator is started at 75% but the text is saying 100%

<img width="539" alt="screen shot 2015-07-22 at 8 55 12 am" src="https://cloud.githubusercontent.com/assets/1656824/8830149/94404502-304f-11e5-86fc-e3fefbce9f3c.png">

FeatureLayer opacity doesn't seem to be working correctly

<img width="548" alt="screen shot 2015-07-22 at 8 55 56 am" src="https://cloud.githubusercontent.com/assets/1656824/8830161/a0ccc610-304f-11e5-90e6-933c0223b13e.png">